src/views/ModelPlan/TaskList/ITSolutions/AddOrUpdateOperationalNeed/index.tsx

```diff
@@ -0,0 +1,328 @@
+                      }}
+                    >
+                      <Fieldset>
+                        <FieldGroup
+                          scrollElement="nameOther"
+                          error={!!flatErrors.nameOther}
+                          className="margin-top-3"
+                        >
+                          <Label htmlFor="it-solution-custom-name-other">
+                            {t('customOperationalNeedName')}
+                            <RequiredAsterisk />
+                          </Label>
+
+                          <FieldErrorMsg>{flatErrors.nameOther}</FieldErrorMsg>
+
+                          <Field
+                            as={TextInput}
+                            error={!!flatErrors.nameOther}
+                            id="it-solution-custom-name-other"
+                            data-testid="it-solution-custom-name-other"
+                            maxLength={50}
+                            name="nameOther"
+                          />
+                        </FieldGroup>
+
+                        <div className="margin-top-6 margin-bottom-3">
+                          {isUpdating ? (
+                            <Button
+                              type="submit"
+                              id="submit-custom-solution"
+                              disabled={!dirty}
+                            >
+                              {t('update')}
+                            </Button>
+                          ) : (
+                            <>
+                              {/* Saves, does not add solution, and returns to tracker */}
+                              <Button
+                                type="button"
+                                id=""
+                                disabled={!values.nameOther}
+                                outline
+                                onClick={() =>
+                                  handleFormSubmit(values, 'it-tracker')
+                                }
+                              >
+                                {t('saveWithoutAdding')}
+                              </Button>
+                              {/* Saves and continues to add solution */}
+                              <Button
+                                type="submit"
+                                id="submit-custom-solution"
+                                disabled={!values.nameOther}
+                              >
+                                {t('continue')}
+                              </Button>
+                            </>
+                          )}
+                        </div>
+
+                        <Button
+                          type="button"
+                          className="usa-button usa-button--unstyled display-flex flex-align-center"
+                          onClick={() => {
+                            history.goBack();
+                          }}
+                        >
```

Comment:
This should not use `goBack()`.  It should return to the task list always, as that is what the text indicates.  It's possible to get here from a different area, and this link doesn't make sense if so.